docker/compose_files/docker-compose.yml

```diff
@@ -214,8 +234,12 @@ services:
       - ${CODALAB_HOME}:${CODALAB_HOME}
       - ${CODALAB_BUNDLE_MOUNT}:${CODALAB_BUNDLE_MOUNT}
       - ${CODALAB_MONITOR_DIR}:${CODALAB_MONITOR_DIR}
+    networks:
+      - rest-server
 
 networks:
+  rest-server:
```

Comment:
I defined a new network called `rest-server`. Any service on this network can simply communicate with the rest server, including the rest server of a second instance.

Reply:
What access does this allow precisely?  Before, the various docker containers (e.g., frontend) could still talk to the rest-server, no?

Reply:
By default, all these containers could talk to each other without explicitly defining `networks` for each one, since a default network gets created for the app. 
I had to add `networks` with the same name for each container in order for all containers   to be able to talk to all other containers in a duplicate set for the second instance. I can try to find a simpler solution too.

Reply:
I think once we define `networks` for one container or service, we override the default network.